pkg/config/vlan.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018-2019 Red Hat, Inc.
+// Copyright 2014 CNI authors
```

Comment:
update copy right to reflect year and nvidia

Reply:
done